source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -29,8 +29,44 @@ Instructor Experiences
+   - After upgrading, if you wish to use the new Content Libraries (beta) or Studio Course Search features, you will need to run ``manage.py cms reindex_studio --experimental --incremental`` to populate the new Studio Search index with your existing content (courseware/libraries). Studio search results will be incomplete until this command has finished.
+- `Course assets are now served by a view rather than a middleware <https://github.com/openedx/edx-platform/issues/34702>`_
+   - Background: The LMS/CMS previously handled course asset requests (asset-v1: andc4x URLs) via a middleware called StaticContentServer. This middleware has been converted to a view.
+   - Action: If your deployment has a custom MIDDLEWARE list in Django, you will need to remove this item at the time of upgrade to Sumac. Otherwise, no action is needed.
+- Ubuntu 22.04 Related Operators Note
+   - `PR <https://github.com/openedx/edx-platform/pull/35450>`_
+   - In newer versions of ubuntu the MD4 hashing algorithm is disabled by default. To enable it the openssl config needs to be updated in a manner similar to what's being done here. Alternatively, you can set the FEATURES['ENABLE_BLAKE2B_HASHING'] setting to True which will switch to a newer hashing algorithm where MD4 was previously used.
+      Because this hashing is being used as a part of the edx-platform caching mechanism, this will effectively clear the cache for the items that use this hash. The will impact any items where the cache key might have been too big to store in memcache so it's hard to predict exactly which items will be impacted.
+- Added override options to commerce related CTA URLs in edx-platform
+   - Background: Extension points have been added have been added to commerce app in: `PR1 <https://github.com/openedx/edx-platform/pull/35441>`_, `PR2 <https://github.com/openedx/edx-platform/pull/35501>` so openedX community members who wants to extend the commerce functionality can do so without explicitly adding code into edx-platform codebase. For more information into extension points see Pluggable override section in `Extension Points <https://github.com/openedx/edx-platform/blob/master/docs/concepts/extension_points.rst>`_.
+   - Additional Considerations: Commerce app itself is slated for deprecation nevertheless.
+   - `courseware.enable_navigation_sidebar <https://github.com/openedx/edx-platform/blob/38f73442e78a8b9afb5543facd170dca830acb1a/lms/djangoapps/courseware/toggles.py#L86>`_  is now enabled by default. It was disabled by default in Redwood.
+
+- New Forums Implementation (migration from Ruby to Python forums backend)
+
+   - For users running Tutor, they will automatically switch to the new app. Instructions to migrate data from mongodb to mysql is available in the plugin README https://github.com/overhangio/tutor-forum/ (not yet merged, see this PR).
```

Comment:
@regisb what PR should this link to?

Reply:
https://github.com/overhangio/tutor-forum/pull/49/files